_includes/codeForm.html

```diff
@@ -121,7 +121,7 @@ <h2>{{ site.LanguagesCodeDesc[page.lang] }}</h2>
     {% assign languages_array = languages_array | uniq  %}
     {% for language in languages_array %}
       <div class="checkbox" >
-        <label><input class="{{ language }}" type="checkbox" value="{{ language }}" name="{{ language }}"> {{ language  }} </label>
+        <label><input class="{{ language }}" datafor="languages" type="checkbox" value="{{ language }}" name="{{ language }}"> {{ language  }} </label>
```

Comment:
If you want to simplify and not use the --ignore-re ..., you can use the accepted syntax for data attributes, which is data-attributename="value" (you just need to add an hyphen, i.e. "data-for") and it should be all good